lib/ecco/ruby_rows.rb

```diff
@@ -0,0 +1,63 @@
+module Ecco
+  module RubyRows
+    ROW = 0
+
+    FIRST_COLUMN  = 0
+    SECOND_COLUMN = 1
+
+    KEY   = 0
+    VALUE = 1
+
+    def self.convert_to_ruby(java_row)
+      if java_row.inspect.include?("LinkedList")
```

Comment:
doesn't `is_a?(....LinkedList)` work?

Reply:
Also not really sure about the naming here, isn't LinkedList vs ArrayList just a consequence of the type of event? So it could make more sense to make a decision based on that rather on type?